packages/ui/src/components/Modal/__stories__/Image.stories.tsx

```diff
@@ -0,0 +1,37 @@
+import type { StoryFn } from '@storybook/react'
+// @ts-expect-error can't import ultraviolet/illustration in ui (cyclic dependencies)
+import { accountExperienceContent } from '@ultraviolet/illustrations/products/accountExperience'
```

Comment:
Better avoid cyclic dependencies. You need to add a folder `assets` at same place as story and add manually an image there. You could also use the same image for unit testing. Check on AvatarV2 how it has been done 😄